src/views/teachers/faq/faq.jsx

```diff
@@ -30,55 +30,151 @@ const TeacherFaq = props => (
+                                    <a href="https://scratch.mit.edu/privacy_policy">
+                                        <FormattedMessage id="teacherfaq.privacyPolicy" />
+                                    </a>
+                                )
+                            }}
+                        />
+                    </dd>
                     <dt><FormattedMessage id="teacherfaq.studentPrivacyLawsTitle" /></dt>
-                    <dd><FormattedMessage id="teacherfaq.studentPrivacyLawsBody" /></dd>
+                    <dd>
+                        <FormattedMessage
+                            id="teacherfaq.studentPrivacyLawsBody"
+                            values={{
+                                privacyPolicyLink: (
+                                    <a href="https://scratch.mit.edu/privacy_policy">
+                                        <FormattedMessage id="teacherfaq.privacyPolicy" />
+                                    </a>
+                                )
+                            }}
+                        />
+                    </dd>
                 </dl>
             </section>
             <section id="community">
                 <span className="nav-spacer" />
                 <h2><FormattedMessage id="teacherfaq.commTitle" /></h2>
                 <dl>
                     <dt><FormattedMessage id="teacherfaq.commHiddenTitle" /></dt>
-                    <dd><FormattedHTMLMessage id="teacherfaq.commHiddenBody" /></dd>
+                    <dd><FormattedMessage id="teacherfaq.commHiddenBody" /></dd>
                     <dt><FormattedMessage id="teacherfaq.commWhoTitle" /></dt>
-                    <dd><FormattedHTMLMessage id="teacherfaq.commWhoBody" /></dd>
+                    <dd><FormattedMessage id="teacherfaq.commWhoBody" /></dd>
+                    <dt><FormattedMessage id="teacherfaq.commTurnOffCommentsTitle" /></dt>
+                    <dd>
+                        <FormattedMessage
+                            id="teacherfaq.commTurnOffCommentsBody"
+                            values={{
+                                desktopLink: (
+                                    <a href="https://scratch.mit.edu/download">
+                                        Scratch Desktop app
+                                    </a>
+                                )
+                            }}
+                        />
+                    </dd>
+                    <dt><FormattedMessage id="teacherfaq.commBlockGamesTitle" /></dt>
+                    <dd><FormattedMessage id="teacherfaq.commBlockGamesBody1" /></dd>
+                    <dd><FormattedMessage id="teacherfaq.commBlockGamesBody2" /></dd>
                     <dt><FormattedMessage id="teacherfaq.commInappropriateTitle" /></dt>
-                    <dd><FormattedHTMLMessage id="teacherfaq.commInappropriateBody" /></dd>
+                    <dd><FormattedMessage id="teacherfaq.commInappropriateBody" /></dd>
```

Comment:
(Same as above) I think you need to keep this as a FormattedHTMLMessage (at least for now, and for a while) because existing translations are going to have HTML in them.